tasks/install_prereqs.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: ensure Auditd is installed
+  package:
+    name: auditd
+    state: present
+
+- name: Add docker gpg key
+  apt_key:
```

Comment:
This doesn't verify that the key is what we expect it to be, and is therefore very unsafe.
Since apt_key doesn't support verifying the downloaded fingerprint, I suggest downloading the key file, verifying the fingerprint matches the documented one ("DD91 1E99 5A64 A202 E859 07D6 BC14 F10B 6D08 5F96") and then adding the key from a copy in this repo rather than grabbing it from t'interwebs every time.

Reply:
will do 👌